src/atoms/Button/Button.web.js

```diff
@@ -370,14 +370,21 @@ const Button = ({
   id,
   name,
 }) => {
-  const theme = useTheme();
-
   if (typeof children !== 'string' && typeof children !== 'undefined') {
     throw new Error(
       `Error in Button: expected \`children\` of type \`string\` but found ${typeof children}.`,
     );
   }
 
+  const theme = useTheme();
+
+  /* Buttons remain focused on chrome after click unless user clicks on any external area.
```

Comment:
Button remains focused on Chrome after click until focused on any other element.
```suggestion
  /* Buttons remain focused on chrome after click unless user clicks on any external area.
```